src/redux/editOptions.js

```diff
@@ -179,7 +179,7 @@ const INITIAL_STATE = {
   currentAction: EditOptions.none,
   hiddenCoins: [],
   isCoinListEdited: false,
-  pinnedCoins: [],
+  pinnedCoins: ['eth'],
```

Comment:
We may need to do the isPinned / pinnedCoins list a bit differently to properly handle when a user has explicitly unpinned ETH

Reply:
this was a preexisting issue via https://github.com/rainbow-me/rainbow/blob/6dce737518e6bfc448742ac2b0e5ece91a431cc7/src/redux/editOptions.js#L28

removed it and fixed!